tests/test_backend.py

```diff
@@ -9,70 +9,85 @@
 import audeer
 
 
+@pytest.fixture(scope='function', autouse=False)
+def tree(tmpdir, request):
+    r"""Create file tree."""
+
+    files = request.param
+    paths = []
+
+    for path in files:
+        if os.name == 'nt':
+            path = path.replace('/', os.path.sep)
+        if path.endswith(os.path.sep):
+            path = audeer.path(tmpdir, path)
+            path = audeer.mkdir(path)
+            path = path + os.path.sep
+            paths.append(path)
+        else:
+            path = audeer.path(tmpdir, path)
+            audeer.mkdir(os.path.dirname(path))
+            path = audeer.touch(path)
+            paths.append(path)
+
+    yield paths
+
+
 @pytest.mark.parametrize(
-    'files, name, folder, version, tmp_root',
+    'tree, archive, files, tmp_root, expected',
     [
-        (
+        (  # empty
+            ['file.ext', 'dir/to/file.ext'],
+            '/archive.zip',
             [],
-            'empty.zip',
-            None,
-            '1.0.0',
             None,
+            [],
         ),
-        (
+        (  # single file
+            ['file.ext', 'dir/to/file.ext'],
+            '/archive.zip',
             'file.ext',
-            'not-empty.zip',
-            None,
-            '1.0.0',
             None,
+            ['file.ext'],
         ),
-        (
+        (  # list
             ['file.ext', 'dir/to/file.ext'],
-            'not-empty.zip',
-            'group',
-            '1.0.0',
+            '/archive.zip',
+            ['file.ext'],
             None,
+            ['file.ext'],
         ),
-        (
+        (  # all files
             ['file.ext', 'dir/to/file.ext'],
-            'not-empty.zip',
-            'group',
-            '2.0.0',
+            '/archive.zip',
+            None,
             'tmp',
+            ['dir/to/file.ext', 'file.ext'],
```

Comment:
Could you add a test here with `files=['file.ext', 'dir/to/file.ext']`. If I remember correctly, we should then have `expected=['file.ext', 'dir/to/file.ext']`.

Reply:
done

Reply:
Not directly you added a test for:
```
files=['dir/to/file.ext', 'file.ext']
expected=['dir/to/file.ext', 'file.ext']
```
but I asked for
```
files=['file.ext', 'dir/to/file.ext']
expected=['file.ext', 'dir/to/file.ext']
```
to see if the order of the provided files has an influence on expected paths.
So maybe just add another test with the reversed list.

Reply:
Ah yes, my bad. I fixed it.